universe/auto_syncer.go

```diff
@@ -184,14 +184,14 @@ func (f *FederationEnvoy) reportErr(err error) {
 // If the sync is successful (even if no diff is generated), then a new sync
 // event will be logged.
 func (f *FederationEnvoy) syncServerState(ctx context.Context,
-	addr ServerAddr) error {
+	addr ServerAddr, syncConfigs SyncConfigs) error {
 
 	log.Infof("Syncing Universe state with server=%v", spew.Sdump(addr))
 
 	// Attempt to sync with the remote Universe server, if this errors then
 	// we'll bail out early as something wrong happened.
 	diff, err := f.cfg.UniverseSyncer.SyncUniverse(
-		ctx, addr, SyncIssuance,
+		ctx, addr, SyncFull, syncConfigs,
```

Comment:
This should pass through the sync type.

Reply:
I think this is ok, as in this instance it isn't directly used for this main call path after all.